druid/src/widget/image.rs

```diff
@@ -190,19 +190,22 @@ impl<T: Data> Widget<T> for Image {
         bc.debug_check("Image");
 
         // If either the width or height is constrained calculate a value so that the image fits
```

Comment:
`Use the fillstrat to determine the size of the image so that the the image will fit perfectly using this filstrat. If it is unconstrained by both width and height take the size of the image.`

Reply:
The problem is that some of the fill strats do take the size of the image but other strategies do not so your alternative is self contradicting.

Reply:
oh I see, I mean the size of the underlying image data.